docker-compose.yaml

```diff
@@ -156,7 +156,6 @@ services:
       - 8001
     volumes:
       - workspace:${WORKSPACE_ROOT}
-      - data:${CONFIG_ROOT}
```

Comment:
Why are we removing the mount?

Reply:
the data volume is only needed for backward compatibility; when users upgrade from an old Airbyte version that relies on file-based configs